web/js/map/ui.js

```diff
@@ -228,12 +228,17 @@ export function mapui(models, config, store, ui) {
     }
     self.selected = self.proj[proj.id];
     const map = self.selected;
-    const currentRotation = proj.id !== 'geographic' && proj.id !== 'webmerc' ? map.getView().getRotation() : 0;
-    store.dispatch({ type: UPDATE_MAP_UI, ui: self, rotation: currentRotation });
+
+    const isProjectionRotatable = proj.id !== 'geographic' && proj.id !== 'webmerc';
```

Comment:
I really like the use of a variable to hold a conditional check that's used more than once here.  Makes things super readable 💯